app/helpers/organisation_helper.rb

```diff
@@ -17,6 +17,14 @@ def organisation_type(organisation)
     end
   end
 
+  def organisation_type_basic(organisation)
+    if organisation.is_a?(SchoolGroup)
+      'trust'
```

Comment:
I think, if you are introducing this helper to return `'trust'`, then you don't need to (or, you need to not) rename the translations from `trust_whatever` to `schoolgroup_whatever`. I may have misunderstood though.

Reply:
Yup, changed it back to trust_whatever lol